solidity/random-beacon/contracts/libraries/Heartbeat.sol

```diff
@@ -66,14 +66,13 @@ library Heartbeat {
         SortitionPool sortitionPool,
         FailureClaim calldata claim,
         Groups.Group storage group,
-        uint256 nonce
+        uint256 nonce,
```

Comment:
We need to update doc above to reflect the function signature.

Reply:
Added docs for members